schedulingApp/unit-tests/test_courses.py

```diff
@@ -122,3 +122,56 @@ def test_validAndExistingCourse(self):
             self.theCourse.getTitle()
         with self.assertRaises(NameError, msg="Semester for deleted course should no longer exist."):
             self.theCourse.getSemester()
+
```

Comment:
Looks good! I'm sure unit tests will be changed to get passing once implemented, but this is a really solid start.